app/controllers/concerns/waste_exemptions_engine/permission_checks.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This concern is responsible for performing permissions checks. It's intended
+# to be included overriden in host applications which have their own specific
+# permissions and roles.
+module WasteExemptionsEngine
+  module PermissionChecks
+    def current_user_can_edit?
+      true
```

Comment:
In these cases I used to: 
```
raise(NotImplementedError) 
```

Reply:
I've done this in the past but was thinking that it was better to leave `true` in as a default - do you think it's better to force host apps to implement their own version of the method? I was thinking it'd be harder to test and manage, but also I can see the benefits of forcing apps to define all their own permissions, although the front-office doesn't really have any concept of them yet...

Reply:
I am concerned about the fact that we might give unauthorised people the ability to change exemptions mostly. What if we raise the exception only in the environments that have the feature turned on then?

Reply:
Editing is currently disabled in the front office so it doesn't feel like a pressing issue – however if we raise a `NotImplementedError` here it will also make our tests more complicated as the dummy app will have to have its own EditHelper config, or we'll have to do a lot more stubbing in the tests themselves. So it feels like more pain than benefit for me at the moment.